src/utils/context_utils.py

```diff
@@ -195,6 +220,27 @@ def get_game_state(context: AgentContext) -> Optional["GameState"]:  # noqa: F82
         return game_state
 
 
+def save_server_settings(server_settings, context: AgentContext):
+    """Save ServerSettings to the KeyValue store."""
+
+    logging.debug(
+        f"Saving Server Settings from workspace {context.client.config.workspace_handle}.",
+        extra={
+            AgentLogging.IS_MESSAGE: True,
+            AgentLogging.MESSAGE_TYPE: AgentLogging.THOUGHT,
+            AgentLogging.MESSAGE_AUTHOR: AgentLogging.AGENT,
+        },
+    )
+
+    # Save it to the KV Store
+    value = server_settings.dict()
+    kv = KeyValueStore(context.client, _SERVER_SETTINGS_KEY)
```

Comment:
I'm curious about the decision to use `KeyValueStore` here, as opposed to e.g. a dedicated file that we know the handle of.  KeyValueStore sets the values as tags, which a) viewing appears to be broken for in the Steamship web UI and b) appears to be a bit clunkier than, for example, having a block with a serialization in it.

Reply:
Both seem perfectly fine to me -- though I believe Files are immutable whereas tags can be deleted -- so that might end up being a practical consideration.

Thanks for the tag-break catch -- I can zip in and post a patch right now!

(cc @dkolas to be sure)